packages/components/stamp/src/stamp.tsx

```diff
@@ -133,17 +150,74 @@ const getStampStyles = (props: StylesFunctionParams) => {
 const Stamp = (props: Props) => {
   const { themedValue } = useTheme();
   const overrideTextColor = themedValue(false, true);
+  const Icon =
+    props.icon &&
+    cloneElement(props.icon, {
+      size: 'medium',
+      color: props.tone ? iconColorsMap[props.tone] : null,
+    });
 
+  if (props.message && props.children) {
+    return (
+      <div
+        css={css`
+          fill: ${designTokens.colorError};
+          color: ${designTokens.colorError};
+        `}
+      >
+        <SpacingsInline alignItems="center">
+          <ErrorIcon />
+          <Text.Detail tone={'critical'}>
```

Comment:
The idea here is not to render anything different to the end user but to let the developer know that we now prefer the `Stamp` component to be used with the inline props.

For some period of time, we want to allow the component used both ways where inline props would have more priority.

I think we don't need this code branch but just replace the `console.log` we're using in the `children` branch by a `warning` as we do in other components ([example](https://github.com/commercetools/ui-kit/blob/main/packages/components/fields/password-field/src/password-field.tsx#L181))

Reply:
Oh yes, that is correct. But this one is in the case where the component is not use as expected. where the user passes the inline prop and at the same time, uses prop.children too. I believe we should give the user some notice that these two cannot be used at the same time. 
something like this: 
```
<Stamp tone={tone} isCondensed={boolean('isCondensed', false)} message={{ id: 'stamp-message', defaultMessage: tone }} icon={<Icon />} >
    <Text.Detail>{`tone="${tone}"`}</Text.Detail>
</Stamp>
```

Reply:
Yes, I understand, but we need to differentiate between a warning for the developer and a warning for the end user.

In case a user provides both the inline props and the children, we would simply ignore the children and use the inline props.
Personally I think this is not an expected use case for a developer so perhaps we don't need to add code for it, but if you feel it's better to catch this scenario, we can use the `warning` utility as we do in other components.
Example:
```js
warning(
  this.message && this.children,
  'Stamp: `children` prop is ignored as `message` was provided and it has more priority'
);
```

Does it make sense to you?

Reply:
Yes this makes sense now. Thank you.

Reply:
Yes I agree with @CarlosCortizasCT , we should only log a warning.

> In case a user provides both the inline props and the children, we would simply ignore the children and use the inline props.

⬆️ That would also be my expectation about the behavior.